test/common/index.js

```diff
@@ -59,6 +59,8 @@ exports.enoughTestCpu = Array.isArray(cpus) &&
                         (cpus.length > 1 || cpus[0].speed > 999);
 
 exports.rootDir = exports.isWindows ? 'c:\\' : '/';
+exports.projectDir = path.resolve(__dirname, '..', '..');
```

Comment:
I think this won't work on our cross-compiling Raspberry Pi devices in CI. @nodejs/build ?

Reply:
Hmmmm...but it *did* pass in the CI, so...hmmm....

Reply:
Oh, probably because CI was run before that specific feature was added maybe? Perhaps? I'm certainly seeing it fail 100% of the time in CI right now as in https://ci.nodejs.org/job/node-test-binary-arm/11032/.

```console
not ok 272 sequential/test-make-doc
  ---
  duration_ms: 2.153
  severity: fail
  stack: |-
    fs.js:926
      return binding.readdir(pathModule.toNamespacedPath(path), options.encoding);
                     ^
    
    Error: ENOENT: no such file or directory, scandir '/home/iojs/build/workspace/node-test-binary-arm/out/doc/api'
        at Object.fs.readdirSync (fs.js:926:18)
        at Object.<anonymous> (/home/iojs/build/workspace/node-test-binary-arm/test/sequential/test-make-doc.js:15:17)
        at Module._compile (module.js:607:30)
        at Object.Module._extensions..js (module.js:618:10)
        at Module.load (module.js:526:32)
        at tryModuleLoad (module.js:489:12)
        at Function.Module._load (module.js:481:3)
        at Function.Module.runMain (module.js:648:10)
        at startup (bootstrap_node.js:191:16)
        at bootstrap_node.js:609:3
  ...
```

Reply:
I wonder if the problem isn't common.projectDir at all but instead that the Pi devices run `make test-ci-js` and not `make test` or `make test-ci`?

Reply:
(...and so it skips the `make doc`....which we probably want it to skip because who knows how long it would take to run on a Pi and we'd be running it on all of them just for a test that only one of them runs...)

Reply:
I think I've got a fix in https://github.com/nodejs/node/pull/16301 but we'll see if CI agrees....

Reply:
Hmm, I wonder why this pass the CI before...anyway thanks for the fix!

Reply:
@joyeecheung I think it passed CI because this change wasn't in the branch when CI ran. It was run before a5262f1 landed.